taskforce/apps/comments/docker-compose.yml

```
@@ -0,0 +1,25 @@
+version: '3.5'
+services:
+  mongodb:
+    image: mongo
+    restart: always
+    container_name: mongodb-comments
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: admin
+      MONGO_INITDB_ROOT_PASSWORD: test
+      MONGO_INITDB_DATABASE: taskforce-comments
+    ports:
+      - 27018:27017
+    volumes:
+      - ./db:/data/db
+
+  mongodb-express:
+    image: mongo-express
+    restart: always
+    container_name: mongo-express-comments
+    ports:
+      - 8082:8081
+    environment:
+      ME_CONFIG_MONGODB_ADMINUSERNAME: admin
+      ME_CONFIG_MONGODB_ADMINPASSWORD: test
+      ME_CONFIG_MONGODB_URL: mongodb://admin:test@mongodb:27018/
```

вот здесь ещё порт не поменял 
```suggestion
      ME_CONFIG_MONGODB_ADMINPASSWORD: test
      ME_CONFIG_MONGODB_URL: mongodb://admin:test@mongodb:27017/
```